explorer/package.json

```diff
@@ -45,11 +48,17 @@
     "whatwg-fetch": "2.0.3"
   },
   "scripts": {
+    "precommit": "npm run lint-staged",
+    "lint-staged": "lint-staged",
+    "pretty": "prettier --write 'src/**/*.js'",
```

Comment:
It should be safe to write `'**/*.js'` here. By default, `prettier` excludes `node_modules`. You can verify this by running `yarn prettier --list-different '**/*.js'` at the command line.